SofaKernel/SofaCommon/SofaCommonConfig.cmake.in

```diff
@@ -3,9 +3,19 @@
 @PACKAGE_INIT@
 
 set(SOFACOMMON_TARGETS @SOFACOMMON_TARGETS@)
+set(SOFAEIGEN2SOLVER_WITH_OPENMP @SOFAEIGEN2SOLVER_WITH_OPENMP@)
 
 find_package(SofaBase REQUIRED SofaComponentBase)
 
+# Eigen3 is required by SofaEigen2Solver and SofaRigid
+find_package(Eigen3 REQUIRED)
+
+# OpenMP might be required by SofaEigen2Solver
+if (SOFAEIGEN2SOLVER_WITH_OPENMP AND NOT TARGET OpenMP::OpenMP_CXX)
```

Comment:
Standard naming is `SOFAEIGEN2SOLVER_HAVE_OPENMP`
Why `AND NOT TARGET OpenMP::OpenMP_CXX` ?

Reply:
Because the FindOpenMP.cmake does not check if the target is already defined and starts to look again, which resulted in a very very slow and verbose cmake call (since a lot of modules depends on SofaCommon, and everyone of them calling find_package(OpenMP) because of this)